src/plugins/vis_type_timeseries/public/application/components/vis_editor_visualization.js

```diff
@@ -194,18 +182,13 @@ class VisEditorVisualizationUI extends Component {
           />
         </EuiFlexItem>
 
-        {panelInterval && (
-          <EuiFlexItem grow={false}>
-            <EuiText color="default" size="xs">
-              <p>
-                <FormattedMessage
-                  id="visTypeTimeseries.visEditorVisualization.panelInterval"
-                  defaultMessage="Interval: {panelInterval}"
-                  values={{ panelInterval }}
-                />
-              </p>
-            </EuiText>
-          </EuiFlexItem>
```

Comment:
It seems like you've actually removed the interval indicator from all charts- can we keep it on all chart types in the editor only?